tasks/inspectors.js

```diff
@@ -28,20 +25,64 @@ var config = require("../config/config"),
       }
     });
 
-function run(options) {
+// load a report from disk, put it into elasticsearch
+function loadReport(details, done) {
+  var base_path = details.base_path,
```

Comment:
See line 105, it's now passed in the object representing the report, along with inspector, year, and report_id.